### PR TITLE
updated fluentd to 1.11.0

### PR DESCRIPTION
**Pull request by monotek**
fixes: https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/240

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,8 +1,8 @@
 PATH
   remote: .
   specs:
-    fluent-plugin-kubernetes_metadata_filter (2.4.6)
-      fluentd (>= 0.14.0, < 1.10)
+    fluent-plugin-kubernetes_metadata_filter (2.4.7)
+      fluentd (>= 0.14.0, < 1.11)
```
Comment (rutkai) [inline Gemfile.lock:5]: This should be `<= 1.11`, isn't it?

File: fluent-plugin-kubernetes_metadata_filter.gemspec [hunk 2]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.1.0'
 
-  gem.add_runtime_dependency 'fluentd', ['>= 0.14.0', '< 1.10']
+  gem.add_runtime_dependency 'fluentd', ['>= 0.14.0', '< 1.11']
```
Comment (rutkai) [inline fluent-plugin-kubernetes_metadata_filter.gemspec:22]: Here as well

File: Gemfile.lock (hunk 1 above)
Comment (monotek) [reply Gemfile.lock]: done

**Comment by jcantrill** [comment]
Bumped the minor version  due to the fluent bump in an alternate PR

**Pull request closed without merging**